internal/store/pod.go

```diff
@@ -221,6 +221,31 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_pod_status_unscheduled_reason",
+			Type: metric.Gauge,
+			Help: "Describes the reason the pod is unschedulable.",
```

Comment:
`Describes the reason the pod is unschedulable.` -> `Describes the reason why the pod is unschedulable.`